website/docs/getting-started/setup/installation-guides/docker/README.md

```diff
@@ -5,172 +5,157 @@ sidebar_position: 1
 
 ## Prerequisites
 
+* [Docker-Compose](https://docs.docker.com/compose/install/) (version 1.29.2 or later) _(Recommended)_
 * [Docker](https://docs.docker.com/get-docker/) (version 20.10.7 or later)
-* [Docker-Compose ](https://docs.docker.com/compose/install/)(version 1.29.2 or later)
 
-Create an installation folder called `appsmith` where you would like your Appsmith installation and data storage.
+1. Create an installation folder called `appsmith` where you would like your Appsmith installation and data storage.
+2. `cd` into the installation folder.
 
-`cd` into the installation folder.
-
-## Quick Start (with docker-compose)
+:::tip
+If you'd like to name your installation directory something other than `appsmith`, update your docker-compose.yml file with the name you'd like:
+```yml
+# docker-compose.yml
 
-:::info
-Appsmith [auto-update](./#enabling-appsmith-auto-updates-with-docker-compose) now also supports auto-backup feature. This backup can be used to roll back an update to a previous version of Appsmith, if the user wants to.
+# ...
+services:
+  appsmith:
+    image: index.docker.io/appsmith/appsmith-ce
+    container_name: <YOUR_DIRECTORY_NAME>
+# ...
+```
 :::
 
-The Appsmith Docker image is built with all the components required to run within a single Docker container. All these multiple processes are managed by a Supervisord instance, which is a lightweight process manager.
-
-:::note
-You can expect one container running when using the `docker-compose ps` command. Something like:
+## Setup with docker-compose
 
-_#Container appsmith Running_
-:::
+The Appsmith Docker image contains all the components required to run within a single Docker container. All these multiple processes are managed by a [Supervisord](http://supervisord.org/) instance, which is a lightweight process manager.
 
 ### Docker-compose configuration
 
-:::info
-Currently, auto-update is disabled on the docker-compose file. If you want to enable auto-update for Appsmith, please uncomment all the commented lines in the docker-compose file.
-:::
+Click the link below to download the `docker-compose.yml` file, and place it into your Appsmith installation folder.
 
-Download the below `docker-compose.yml` file into the appsmith installation folder
-
-<a target="_blank" href='/img/docker-compose_(3).yml' download="docker-compose_(3).yml">docker-compose.yml (Click to Download)</a>
+<div class="downloadAssets">
+    <a target="_blank" href='/img/docker-compose_(3).yml' download="docker-compose_(3).yml">
+     <img src="/img/FileDownload.png" alt="Click to Download"/>
+    </a>
+</div>
 
-**or** run the following curl if you're on a remote machine:
+**OR,** run the following cURL:
 
 ```bash
 curl -L https://bit.ly/3AQzII6 -o $PWD/docker-compose.yml
 ```
 
-:::note
-For the **Business Edition** - change the **image** name from `appsmith-ce` to `appsmith-ee` for the `image:` key in the `docker-compose.yml` file.
-:::
+:::info
+**For Business Edition users:**
```

Comment:
It is important for business users -- it exists within an info callout b/c the `:::info` stands out well on the page, and it can be simply inserted at the point in the steps where a business user needs to take one extra step in setup.

I can remove the styling if you like, just let me know -- I updated the bold text to `Business Edition`